### ISSUE TITLE
[Feature Request] Add an export as curl button

### ISSUE THREAD

**Issue by azgaviperr** [frikky/Shuffle#370]
It would be great to export the command sent on an execution as curl command for debugging purpose. 
Maybe take away all authent part ?

**Comment by frikky** [comment]
Yep - the python code is not available through the API right now, so this is a larger job than it may seem. This goes along the same lines of export HTTP app as app as well

The authentication part isn't available to the user (ever) either, so it would be something like "REPLACEME", with the URL being a part of the curl request.
